src/Symfony/Component/Serializer/CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ CHANGELOG
 * added optional `int[] $ignoredNodeTypes` argument to `XmlEncoder::__construct`. XML decoding now
   ignores comment node types by default.
 * added `ConstraintViolationListNormalizer`
+* removed redundant `@requires PHP 7.1` from `AbstractNormalizerTest.php`
```

Comment:
can you remove this line please? it's really not useful to pass this info to the reader :)

Reply:
Sure

Reply:
Just pushed the modification. Yay. I deleted three lines.